CHANGELOG.md

```diff
@@ -19,9 +19,18 @@ Remember to align the itemized text with the first line of an item within a list
   * {func}`jax.scipy.stats.mode` now returns a 0 count if the mode is taken
     across a size-0 axis, matching the behavior of `scipy.stats.mode` in SciPy
     1.11.
+  * Removed the "No GPU/TPU" found warning. Instead warn if, on Linux, an
```

Comment:
can you add the cuda 12.2 thing too please?

Reply:
Done.